.devcontainer/dockerBuild.sh

```diff
@@ -0,0 +1,14 @@
+#environment setup
+sudo chmod -R a+rwx /etc/bash.bashrc
+echo "export PATH=\"/home/node/.cache/pypoetry/virtualenvs/gbstats-vabhrsvx-py3.7/bin:$PATH:$HOME/.poetry/bin\"" >> /etc/bash.bashrc
```

Comment:
This cache path doesn't look very stable.  Can we get this from poetry dynamically instead and/or use the poetry `activate` method?

Reply:
Also, do we need to use the global .bashrc file?  It would be much simpler if we could use one in the user's home directory (won't need to mess with permissions, will be able to `source` it in the current shell, etc.)